x/ecosystem-incentive/types/expected_keepers.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/auth/types"
+)
+
+// AccountKeeper defines the expected account keeper used for simulations (noalias)
+type AccountKeeper interface {
```

Comment:
I think AccountKeeper is not used and this could be removed.

Reply:
forgot to do this, thanks.
successfully removed now.